### ISSUE TITLE
SDK add same header and footer as .com + GH stars

### ISSUE THREAD

**Issue by emilylucie** [meltano/sdk#895]
<img width="1380" alt="Screen Shot 2022-08-08 at 3 44 22 PM" src="https://user-images.githubusercontent.com/84082383/183511225-8861aa48-6418-4f57-bfa2-66af75927fb6.png">
<img width="1388" alt="Screen Shot 2022-08-08 at 3 44 35 PM" src="https://user-images.githubusercontent.com/84082383/183511227-a68d75f2-96c6-406e-8551-e18875b4b50c.png">

as discussed in the leadership call we agreed to look into adding consistent heads across all our web entities (.com, sdk, hub, docs, maybe handbook). 

### Why do this:

- User experience 
- No dead ends- lost opportunity
- Consistency + branding
- Give people a clearer picture of the Meltano ecosystem 

My suggestion would be to start by just adding the header and footer from the .com pages (above) if that is even possible. If this is not possible, I would create a paired-down version for this purpose. 

### Must haves at a min:
- always linking back to home page
- always has star us- link to github 
- slack link
- social links are a nice to have 
-
<img width="528" alt="Screen Shot 2022-08-08 at 3 52 24 PM" src="https://user-images.githubusercontent.com/84082383/183512431-3dfcc14d-a460-40d3-89b7-8f6bc71f2a3f.png">

**Comment by emilylucie** [comment]
https://github.com/meltano/hub/issues/709

**Comment by emilylucie** [comment]
https://github.com/meltano/internal-general/issues/436

**Comment by tayloramurphy** [comment]
@aaronsteers @alexmarple related to the other header/footer one - what's the complexity on this? It's a completely different theme but we still want some consistency.

**Comment by aaronsteers** [comment]
@tayloramurphy and @emilylucie - I think this is relatively low complexity (probably a`4` weight) but there's too much left missing in the spec, which we need to resolve before we start.

The header has things that don't really make sense for the SDK site and the Footer likewise feels odd for the SDK context.

**Comment by aaronsteers** [comment]
E.g. - if we replace "Hub" with "SDK", does this meet requirements for footer?

<img width="800" alt="image" src="https://user-images.githubusercontent.com/18150651/185022183-650c222b-091b-480d-a244-eb586f26ec4f.png">

For the header, should we start with what we have in https://docs.meltano.com below and remove all the headers (that would be, removing `Home  |  Guides  |  Concepts  |  Tutorials  |  Reference  |  Contribute  |  Get Help` - since all of those titles have an incorrect inference of where they would land releative to the SDK.)

<img width="1210" alt="image" src="https://user-images.githubusercontent.com/18150651/185022542-202928f2-7796-40ec-96c9-0aab1308d952.png">

Removing all the headers seems to leave the rest quite bare - and might not be appropriate with the all-blue and same-colored left-hand sidebar.

**Comment by aaronsteers** [comment]
Another approach that might be less disruptive would be to keep the overall layout we have today...

<img width="1208" alt="image" src="https://user-images.githubusercontent.com/18150651/185023322-2bb4cec9-ade9-41fd-8c89-1c38e66264ee.png">

...

<img width="1209" alt="image" src="https://user-images.githubusercontent.com/18150651/185023365-aec5de3e-a017-474d-a579-b7003af33d64.png">

And fit these requirements into the existing layout (adapted from @emilylucie's comment above):

- linking back to home page in top-left header
- star us-link in top-right (sdk repo though, not meltano, yes?)
- slack link in bottom-right footer
- social links in bottom-right footer, modeled after Hub footer

**Comment by tayloramurphy** [comment]
@aaronsteers I chatted with @afolson and she'll be taking the Docs and SDK one. So we'll just need to have Alex focus on https://github.com/meltano/hub/issues/709 which, for the most part I think we can do a drop-in of the current state from the main Meltano site and take a few iterations from there.
